datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -14,27 +14,50 @@ The next step is creating a backup policy and disaster recovery plan.
 ## Backup policy
 
 As part of your backup policy you should regularly create backups of UCP.
-To create a backup of UCP, use the `docker/ucp backup` command. This creates
-a tar archive with the contents of the [volumes used by UCP](../architecture.md)
-to persist data, and streams it to stdout.
 
-You need to run the backup command on a UCP manager node. Since UCP stores
-the same data on all manager nodes, you only need to create a backup of a
-single node.
+To create a UCP backup, you may use the `{{ page.docker_image }} backup` command
```

Comment:
This is way more complete, but you sound like a boring lecturer in some sentences. Try writing like you speak.

What about

To create a backup of UCP, you can run the `{{ page.docker_image }} backup` command on a UCP manager node. <you don't need to tell me you're going to explain something below. If it's interesting I'm going to continue reading>
This command creates a tar archive...